manifests/d/DominikReichl/KeePass/2.49.0/DominikReichl.KeePass.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DominikReichl.KeePass
+PackageVersion: 2.49.0
+PackageLocale: en-US
+Publisher: Dominik Reichl
+PublisherUrl: https://keepass.info
+PublisherSupportUrl: https://sourceforge.net/p/keepass/bugs/
+PrivacyUrl: 'https://keepass.info/help/base/terms.html#privacy'
+Author: Dominik Reichl
+PackageName: KeePass
+PackageUrl: https://keepass.info
+License: GNU GPLv2
+LicenseUrl: https://keepass.info/help/v2/license.html
+Copyright: Copyright © 2003-2021 Dominik Reichl.
+CopyrightUrl: 'https://keepass.info/help/base/terms.html#copy'
+ShortDescription: KeePass, the free, open source, light-weight and easy-to-use password manager.
+Description: KeePass Password Safe is a free, open source, lightweight, and easy-to-use password manager for Windows, Linux and Mac OS X, with ports for Android, iPhone/iPad and other mobile devices. With so many passwords to remember and the need to vary passwords to protect your valuable data, it’s nice to have KeePass to manage your passwords in a secure way. KeePass puts all your passwords in a highly encrypted database and locks them with one master key or a key file. As a result, you only have to remember one single master password or select the key file to unlock the whole database. And the databases are encrypted using the best and most secure encryption algorithms currently known, AES and Twofish. See our features page for details.
+Moniker: keepass
+Tags:
+- password
+- password manager
+- password safe
+- credentials
+- protect
+- cross-platform
+- foss
```

Comment:
```suggestion
Tags:
- password
- password-manager
- password-safe
- credentials
- protect
- cross-platform
- foss
```

Reply:
These tags are as they were in the [previous version of this file](https://github.com/microsoft/winget-pkgs/blob/master/manifests/d/DominikReichl/KeePass/2.48.1/DominikReichl.KeePass.locale.en-US.yaml). I'm not keen on changing them without some specific reason.

Reply:
The best practice is to write all tags in lower case and replace spaces with hyphens.

Reply:
@vedantmgoyal2009 okay, following best practice is potentially a reason to change things. Where is this specified as best practice? I can see [the manifest spec](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md) has a section on best practices for manifest files, and it doesn't seem to be specified there, nor in any of the other documentation I've read through.

I can see using hyphens might be _common_ practice, but it's clearly not a universal practice. If it were actively desirable to format tags this way, it would be fairly straightforward to add checking that all tags are lower case and have no spaces to the manifest validation tests, and to write a script to update the manifests for the many packages that don't do that. That clearly hasn't happened, from which I can only conclude it isn't a requirement.

Without an explicit statement from the Windows Package Manager folks about how they want to handle tags or define best practices, I'm reluctant to change someone else's work just because other people have done things a different way. In particular, I don't think it's necessarily the case that increasing consistency is better than minimising changes.

Reply:
You can see here: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags

Reply:
Perhaps more constructively: if this is (or should be) best practice, I expect the correct thing to do would be to raise a new issue to discuss this, get agreement from the folks running this project. Once there's agreement that it's the right way to do things, someone (you?) could generate a pull request documenting that it's best practice, and ideally fixing up the existing manifest files and/or adding a check to the CI validation tests.

Reply:
Gotcha! Thank you! I'd clearly managed to miss that line.

Reply:
@vedantmgoyal2009 and in the name of following my own advice, I've just raised #27577.

Reply:
Thanks a lot. I'm pretty sure that even small changes will help in making the end user's experience better.